evap/evaluation/urls.py

```diff
@@ -12,5 +12,6 @@
     path("contact", views.contact, name="contact"),
     path("key/<int:key>", views.login_key_authentication, name="login_key_authentication"),
     path("profile", views.profile_edit, name="profile_edit"),
+    path("set_notebook", views.notebook, name="notebook_edit"),
```

Comment:
three names "set_notebook", "notebook" and "notebook_edit". Can we unify?

Reply:
ok